runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -280,10 +286,37 @@ public void onConfigChanged(final ConfigChanged event)
 					panel.setFilterMode(config.subscriptionFilter());
 					updateList();
 					break;
+				case "customWorldCycle":
+					setCustomWorldCycle();
+					break;
 			}
 		}
 	}
 
+	private void setCustomWorldCycle()
+	{
+		String worldList = config.customWorldCycle();
+		WorldResult worldResult = worldService.getWorlds();
+		worldCycleList.clear();
```

Comment:
Run as little code as you can early on. In this case, I would move the `!worldList.isEmpty()` check from the following `if` statement up here to become a guard clause.
```suggestion
		worldCycleList.clear();

		String worldList = config.customWorldCycle();
		if (worldList.isEmpty())
		{
			return;
		}
		
		WorldResult worldResult = worldService.getWorlds();
```

Reply:
For consistency, should I also have an early return on the following worldResult null check? its effectively the same as nothing else is checked past said point.

Reply:
For the last block of code in a method, it matters less, but the way you've changed it is fine.